e2e/keywords/setting.resource

```diff
@@ -4,8 +4,29 @@ Documentation    Setting Keywords
 Library    ../libs/keywords/setting_keywords.py
 
 *** Keywords ***
-Set setting ${setting_key} to ${setting_value}
-    update_setting    ${setting_key}    ${setting_value}
-
 Set setting ${setting_key} to ${setting_value} will fail
-    Run Keyword And Expect Error    *    update_setting    ${setting_key}    ${setting_value}    retry=False
\ No newline at end of file
+    [Arguments]    ${retry}=False
+    Run Keyword And Expect Error    *    update_setting    ${setting_key}    ${setting_value}    retry=${retry}
```

Comment:
No need to use `[Arguments]` block here because this keyword doesn't accept arbitrary arguments. `update_setting retry=False` should be good enough.